pkg/catalog/manager/traverse.go

```diff
@@ -20,6 +24,10 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 )
 
+type VersionCommits struct {
+	Value map[string]string `json:"Value,omitempty"`
+}
```

Comment:
Moved out of types since this field does not directly correspond to a CRD field.